### PR TITLE
Feature/conduit support

### PR DESCRIPTION
**Pull request by kschrief**

### CODE REVIEW

File: server/src/messageHandler.mjs [hunk 1]
```diff
@@ -108,16 +109,16 @@ async function handleMessage(message, userId, ws) {
   }
 
   // Fire pre trigger if there is one for this method  
-  if ( oMsg.method in methodTriggers ) {
-    if ( 'pre' in methodTriggers[oMsg.method] ) {
+  if (oMsg.method in methodTriggers) {
```
Comment (michaelcfine) [inline server/src/messageHandler.mjs:112]: Please don't go out of your way to change the style of existing code. It's fine as-is, but the code base uses spaces within parens in conditionals and I don't see a good reason to change. Thanks.

File: server/src/messageHandler.mjs [hunk 2]
```diff
@@ -108,16 +109,16 @@ async function handleMessage(message, userId, ws) {
...
+    if ('pre' in methodTriggers[oMsg.method]) {
       try {
         const ctx = {
           logger: logger,
           setTimeout: setTimeout,
           setInterval: setInterval,
           set: function ss(key, val) { return stateManagement.setScratch(userId, key, val) },
           get: function gs(key) { return stateManagement.getScratch(userId, key); },
-          sendEvent: function(onMethod, result, msg) {
+          sendEvent: function (onMethod, result, msg) {
```
Comment (michaelcfine) [inline server/src/messageHandler.mjs:121]: This whitespace, while fine, shouldn't be here.

**Review by michaelcfine — APPROVED** [review]
Approved, but see comments about unnecessary changes to the style of the code. Please leave existing style as-is.

File: server/src/stateManagement.mjs [hunk 3]
```diff
@@ -116,6 +116,18 @@ async function getAppropriateDelay(userId, methodName) {
   return dly;
 }
 
+function hasOverride(userId, methodName) {
```
Comment (vyvanand) [inline server/src/stateManagement.mjs:119]: this method already exists. we can remove this.

Comment (kschrief) [reply server/src/stateManagement.mjs:119]: Addressed

**Review by michaelcfine — APPROVED** [review]
👍 I think this still looks good.

**Review by vyvanand — APPROVED** [review]

**Review by dhillomk — APPROVED** [review]

**Pull request merged**